test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Error.Tests.ps1

```diff
@@ -26,6 +26,17 @@ Describe 'Get-Error tests' -Tag CI {
 
         $out = Get-Error | Out-String
         $out | Should -BeLikeExactly '*InnerException*'
+
+        $err = Get-Error
+        $err | Should -BeOfType [System.Management.Automation.ErrorRecord]
+        $err.PSObject.TypeNames | Should -Not -Contain 'System.Management.Automation.ErrorRecord'
+        $err.PSObject.TypeNames | Should -Contain 'System.Management.Automation.ErrorRecord#PSExtendedError'
+
+        # need to exercise the formatter
```

Comment:
you might elaborate on this comment. Casual inspection looks like you're just testing the same thing twice.

Reply:
Will expand comment